CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ Versioning](http://semver.org/spec/v2.0.0.html).
   (https://github.com/spotahome/redis-operator) and it must be installed
   on your OpenShift cluster before deploying.
 
+### Removed
+
+- probes on edxapp workers. They are too slow to be run
```

Comment:
This is not true: the real problem is that existing probes are not compatible with redis sentinel and alternative probes based on Django management commands or Celery using Django settings are too slow to be considered as a relevant alternative. You must add that this is a temporary removal until we implement a new efficient probe.